### PR TITLE
[OSF-4581]

### PR DESCRIPTION
**Pull request by MerlinZhang**
Email notifications for comments/file uploads now contain a link to user settings.

### CODE REVIEW

File: website/templates/emails/notify_base.mako [hunk 1]
```diff
@@ -43,7 +43,7 @@
                 <tbody>
                     <tr>
                         <td style="border-collapse: collapse;">
-                            <p class="small text-center" style="text-align: center;font-size: 12px;">To change your email notification preferences, visit your project or user settings.</p>
+                            <p class="small text-center" style="text-align: center;font-size: 12px;">To change your email notification preferences, visit your project or <a href="https://osf.io/settings/notifications/">user settings</a>.</p>
```
Comment (samanehsan) [inline website/templates/emails/notify_base.mako]: This url will break on staging. Instead of hard-coding the `https://osf.io/` can you use `settings.DOMAIN` (from `website.settings`)?

**Comment by samanehsan** [comment]
Also, something to keep in mind for future PRs -- It's helpful to have commit messages that describe what was changed in the commit: http://cosdev.readthedocs.org/en/latest/style_guides/git.html?highlight=commit

**Comment by samanehsan** [comment]
Pass complete @MerlinZhang! :cookie:

File: website/settings/defaults.py [hunk 2]
```diff
@@ -296,7 +296,7 @@ def parent_dir(path):
     CELERYBEAT_SCHEDULE = {
         '5-minute-emails': {
             'task': 'notify.send_users_email',
-            'schedule': crontab(minute='*/5'),
+            'schedule': crontab(minute='*/5 '),
```
Comment (chennan47) [inline website/settings/defaults.py]: Is this change by purpose? Or just mistake typing?

File: website/templates/emails/notify_base.mako [hunk 3]
```diff
@@ -43,7 +46,7 @@
                 <tbody>
                     <tr>
                         <td style="border-collapse: collapse;">
-                            <p class="small text-center" style="text-align: center;font-size: 12px;">To change your email notification preferences, visit your project or user settings.</p>
+                            <p class="small text-center" style="text-align: center;font-size: 12px;">To change your email notification preferences, visit your project or <a href=${website.settings.DOMAIN + "settings"}>user settings</a>. </p>
```
Comment (chennan47) [inline website/templates/emails/notify_base.mako]: Please check the style guide and avoid inline style.

File: website/templates/emails/notify_base.mako [hunk 4]
```diff
@@ -43,7 +46,7 @@
                 <tbody>
                     <tr>
                         <td style="border-collapse: collapse;">
-                            <p class="small text-center" style="text-align: center;font-size: 12px;">To change your email notification preferences, visit your project or user settings.</p>
+                            <p class="text-smaller text-center" style="text-align: center;font-size: 12px;">To change your email notification preferences, visit your project or <a href=${website.settings.DOMAIN + "settings"}>user settings</a>. </p>
```
Comment (samanehsan) [inline website/templates/emails/notify_base.mako]: Since you only use `settings` from `website`, it would be better to only import `settings` (instead of the entire website module)

File: website/templates/emails/notify_base.mako [hunk 5]
```diff
@@ -56,7 +59,7 @@
                     <tbody>
                         <tr>
                             <td style="border-collapse: collapse;">
-                                <p class="small text-center" style="text-align: center;font-size: 12px;">Copyright &copy; 2015 Center For Open Science, All rights reserved. </p>
+                                <p class="text-smaller text-center" style="text-align: center;font-size: 12px;">Copyright &copy; 2015 Center For Open Science, All rights reserved. </p>
```
Comment (samanehsan) [inline website/templates/emails/notify_base.mako]: Why change this CSS class?

**Comment by samanehsan** [comment]
Pass complete @MerlinZhang! :fallen_leaf:

File: website/templates/emails/notify_base.mako [hunk 6]
```diff
@@ -43,7 +46,7 @@
                 <tbody>
                     <tr>
                         <td style="border-collapse: collapse;">
-                            <p class="small text-center" style="text-align: center;font-size: 12px;">To change your email notification preferences, visit your project or <a href="http://osf.io/settings/notifications/">user settings</a>.</p>
+                            <p class="text-smaller text-center" style="text-align: center;font-size: 12px;">To change your email notification preferences, visit your project or <a href=${settings.DOMAIN + "settings"}>user settings</a>. </p>
```
Comment (samanehsan) [inline website/templates/emails/notify_base.mako]: I think this should link to `settings/notifications/` to take the user directly to where they can change their email preferences.

**Comment by MerlinZhang** [comment]
Failing test not related to code changes

File: website/templates/emails/notify_base.mako [hunk 7]
```diff
@@ -56,9 +59,8 @@
                     <tbody>
                         <tr>
                             <td style="border-collapse: collapse;">
-                                <p class="small text-center" style="text-align: center;font-size: 12px;">Copyright &copy; 2015 Center For Open Science, All rights reserved.
-                                ${self.footer()}
-                                </p>
+                                <p class="text-smaller text-center" style="text-align: center;font-size: 12px;">Copyright &copy; 2015 Center For Open Science, All rights reserved. </p>
+                                ${footer()}
```
Comment (sloria) [inline website/templates/emails/notify_base.mako]: This appears incorrect; Why was this changed from `self.footer()`?

File: website/templates/emails/notify_base.mako [hunk 8]
```diff
@@ -43,7 +46,7 @@
                 <tbody>
                     <tr>
                         <td style="border-collapse: collapse;">
-                            <p class="small text-center" style="text-align: center;font-size: 12px;">To change your email notification preferences, visit your project or <a href="http://osf.io/settings/notifications/">user settings</a>.</p>
+                            <p class="text-smaller text-center" style="text-align: center;font-size: 12px;">To change your email notification preferences, visit your project or <a href=${settings.DOMAIN + "settings/notifications/"}>user settings</a>. </p>
```
Comment (sloria) [inline website/templates/emails/notify_base.mako]: I think you will need to put quotation marks around the `href` to make it valid HTML. You should verify this in the email output.

**Comment by sloria** [comment]
Pass finished. There's a lot of noise in the history for this PR. @MerlinZhang to clean up the history and avoid unintended changes to develop, can you please send a new PR containing only your changes?
